pkgs/applications/virtualization/singularity/packages.nix

```diff
@@ -0,0 +1,63 @@
+{ lib
+, callPackage
+, fetchFromGitHub
+}:
+
+{
+  singularity-legacy = (callPackage ./generic.nix rec {
+    pname = "singularity-legacy";
+    version = "3.8.7";
+    projectName = "singularity";
+
+    src = fetchFromGitHub {
+      owner = "apptainer";
+      repo = "singularity";
+      rev = "v${version}";
+      sha256 = "7NFAJpSxtWRGNU6dhxr5bnLTZbXhpTxt0NBvteG2pbQ=";
+    };
+
+    # Update by running
+    # nix-prefetch -E "{ sha256 }: ((import ./. { }).singularity-legacy.override { vendorSha256 = sha256; }).go-modules"
+    # at the root directory of the Nixpkgs repository
```

Comment:
Doesn't nix-update work?

Reply:
I haven't heard about that. Sound's cool!

Reply:
It doesn't seem to work here.